php/EE/Model/Base.php

```diff
@@ -7,7 +7,7 @@
 /**
  * Base EE Model class.
  */
-abstract class Base {
+abstract class Base extends \ArrayObject {
```

Comment:
This was needed for site type PHP and wp where some existing methods were called. Those methods were called in site create too where they use array index to access properties. In site update, we use object property accessor to get the values.